src/pages/WishlistPage.js

```diff
@@ -8,36 +10,31 @@ export default function WishlistPage({
 }) {
   return (
     <>
-      <header>
-        <h1>Diving Wishlist</h1>
-      </header>
-      <main>
-        <Wrapper>
-          {diveWishes.length > 0 ? (
-            diveWishes.map(wish => (
-              <DiveWish
-                key={wish.id}
-                id={wish.id}
-                destination={wish.destination}
-                notes={wish.notes}
-                isBookmarked={wish.isBookmarked}
-                onToggleBookmark={() => onToggleBookmark(wish.id)}
-                onDeleteDiveWish={onDeleteDiveWish}
-              />
-            ))
-          ) : (
-            <Message>
-              You currently have nothing on your wishlist. Start by adding some
-              destinations you would like to dive.
-            </Message>
-          )}
-        </Wrapper>
-      </main>
+      <Heading>Diving Wishlist</Heading>
+      <Grid>
+        {diveWishes.length > 0 ? (
+          diveWishes.map(wish => (
+            <DiveWish
+              key={wish.id}
+              destination={wish.destination}
+              notes={wish.notes}
+              isBookmarked={wish.isBookmarked}
+              onToggleBookmark={() => onToggleBookmark(wish.id)}
+              onDeleteDiveWish={() => onDeleteDiveWish(wish.id)}
+            />
+          ))
+        ) : (
+          <Message>
```

Comment:
You could think about creating an extra component for your empty state message, since you are using it twice and maybe more often in the future

Reply:
Yeah, that might be an idea. Will have a look.